### ISSUE TITLE
OKD 4.15 Community and Certified Operators issue

### ISSUE THREAD

**Issue by mailman-2097** [okd-project/okd#2033]
**Describe the bug**
OKD OperatorHub is not operational as no operators are being displayed
![okd-operatorhub-issue](https://github.com/user-attachments/assets/3afe9053-4b22-43c4-a46d-b17a9d2f884e)

```

oc logs -f community-operators-cwqn6 -n openshift-marketplace -c extract-utilities
cp: can't stat '/bin/copy-content': No such file or directory

oc logs -f community-operators-cwqn6 -n openshift-marketplace -c extract-content
Error from server (BadRequest): container "extract-content" in pod "community-operators-cwqn6" is waiting to start: PodInitializing

oc logs -f community-operators-cwqn6 -n openshift-marketplace -c registry-server
Error from server (BadRequest): container "registry-server" in pod "community-operators-cwqn6" is waiting to start: PodInitializing

--- 

oc logs -f certified-operators-fm5gn -n openshift-marketplace -c registry-server
Error from server (BadRequest): container "registry-server" in pod "certified-operators-fm5gn" is waiting to start: PodInitializing

oc logs -f certified-operators-fm5gn -n openshift-marketplace -c extract-content
Error from server (BadRequest): container "extract-content" in pod "certified-operators-fm5gn" is waiting to start: PodInitializing

oc logs -f certified-operators-fm5gn -n openshift-marketplace -c extract-utilities
cp: can't stat '/bin/copy-content': No such file or directory

```

**Version**
This is OKD Single Node Cluster created with UPI install on Proxmox VE running on HP MiniPC i7-10700T CPU @ 2.00GHz, 32GB Ram, NVME SSD.

```
ClusterID: 1a5fe626-2e80-4e0c-bf38-ac7e7708c275
ClientVersion: 4.15.0-0.okd-2024-03-10-010116
ClusterVersion: Stable at "4.15.0-0.okd-2024-03-10-010116"
ClusterOperators:
        clusteroperator/cloud-credential is missing
        clusteroperator/image-registry is missing
        clusteroperator/operator-lifecycle-manager is missing
```

**How reproducible**
This should be reproducible easily.

**Log bundle**
https://www.dropbox.com/scl/fi/9ji7cf1z50z01u18xt6s3/must-gather.local.3668016615067012211.tar.gz?rlkey=5lwqwxlftecdtr3r3esld4dqj&st=gn19kf9w&dl=0

**Comment by JaimeMagiera** [comment]
Hi, 

This is due to an upstream change by Red Hat. If you remove the existing community operators catalogsource and add this one, you should be good. 

`apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: community-operators  
  namespace: openshift-marketplace
spec:
  displayName: Community Operators
  image: quay.io/operatorhubio/catalog:latest
  publisher: operatorhubio
  sourceType: grpc  `

Not sure if you’ve seen this or not…

https://okd.io/blog/2024/06/01/okd-future-statement/

In short, we’ve shifted focus to OKD 4.16 SCOS and greater. We’re working on testing and documenting a clear path from 4.15 FCOS to 4.16 SCOS and then 4.17 SCOS. Stay tuned for that.

**Comment by mailman-2097** [comment]
Many thanks for your response..

**Comment by mailman-2097** [comment]
I was able to setup the okd 4.16. The operator hub is working now.
